RevenueCatUI/CustomerCenter/Data/CustomerCenterConfigData+Mock.swift

```diff
@@ -125,23 +127,15 @@ enum CustomerCenterConfigTestData {
                 email: "test-support@revenuecat.com",
                 shouldWarnCustomerToUpdate: shouldWarnCustomerToUpdate,
                 displayPurchaseHistoryLink: displayPurchaseHistoryLink,
-                shouldWarnCustomersAboutMultipleSubscriptions: false
+                shouldWarnCustomersAboutMultipleSubscriptions: shouldWarnCustomersAboutMultipleSubscriptions
             ),
             lastPublishedAppVersion: lastPublishedAppVersion,
             productId: 1
         )
     }
 
     @available(iOS 14.0, *)
-    static let customerCenterData = customerCenterData(lastPublishedAppVersion: "1.0.0")
-
-    static let standardAppearance = CustomerCenterConfigData.Appearance(
-        accentColor: .init(light: "#007AFF", dark: "#007AFF"),
-        textColor: .init(light: "#000000", dark: "#ffffff"),
-        backgroundColor: .init(light: "#f5f5f7", dark: "#000000"),
-        buttonTextColor: .init(light: "#ffffff", dark: "#000000"),
-        buttonBackgroundColor: .init(light: "#287aff", dark: "#287aff")
-    )
+    static let `default` = mock(lastPublishedAppVersion: "1.0.0")
```

Comment:
Nit: since `lastPublishedAppVersion` parameter already defaults to `"1.0.0"`, I believe this could be just 
```suggestion
    static let `default` = mock()
```